.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -5,11 +5,12 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking the time to fill out this bug report. Please take the time to provide as much details as possible. Following our [bug report guidelines](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) will help maintainers qualify your report faster.
-
-        Reporting bugs with "error 500" or "white page" may not help us reproduce them. This is a universal error that hides what is underneath. Instead, enable debug mode in the Performance section of PrestaShop and try again. You should get a proper error message.
-
-        🇬🇧 Use English, reports in all other languages will be closed.
+         #### Basic information ❗
+         - **Provide as much detail as possible** - error logs, screenshots, your exact configuration. If the issue cannot be reproduced, it cannot be fixed.
+         - Follow the [bug report guidelines](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) to help issue managers qualify your report faster.
+         - Avoid reporting "error 500" or "white page" errors - this is a universal error message that does not provide enough information to qualify the issue. Enable debug mode in the Performance section of PrestaShop or manually in `/config/defines.inc.php` and try again. You should get a proper error message.
+         - **Write in English.** Reports in all other languages will be closed.
+         - If reporting a problem with the upgrade process, open `/admin/autoupgrade/tmp` and attach anonymized `log.txt` to the issue. It contains all the information about the upgrade process.
```

Comment:
```suggestion
         - If reporting a problem with the upgrade process, open `/admin/autoupgrade/tmp` and attach the last part of `log.txt` to the issue. It may contain information about the upgrade errors helpful in resolving the issue.
```

Reply:
@Hlavtox wdyt?

Reply:
@kpodemski https://www.youtube.com/watch?v=Jo6fKboqfMs